EventDisplay/src/DataInterface.h

```diff
@@ -98,6 +97,8 @@ class DataInterface
 
   std::unique_ptr<mu2e::ParticleInfo> _particleInfo;
 
+  art::Event *_event;
+
```

Comment:
Is access to the event needed inside callbacks there  registered with ROOT and called from ROOT?

If not, this is a little scary.  Is it possible to pass the event as an argument down the call chain? 

When you stash a pointer to the event there is always the danger that the pointer will still be there when you are processing the next event.  Unless there is a structural reason outside of our control I strongly recommended that you pass the event as an argument and never cache a pointer to it.  I am sure that you know what your are doing but people might copy the code; and you might come to it after a long hiatus and forget when the pointer is valid and when it is not valid.

Reply:
I will change this part.

Reply:
I pushed the change.